packages/linux-drivers/RTL8192EU/package.mk

```diff
@@ -17,7 +17,7 @@
 ################################################################################
 
 PKG_NAME="RTL8192EU"
-PKG_VERSION="a7fd035"
```

Comment:
The current hash in master is `dcd71fa` after the 4.11 kernel bump commits were merged so you need to rebase the change. Also, have you done any testing against 8.0? - i'm wondering if we should update drivers there too?